pages/interfaces.json

```diff
@@ -55,5 +55,11 @@
   },
   "docs-ui.concepts": {
     "component": "ConceptsList"
+  },
+  "no-wrapper": {
+    "around": []
+  },
+  "no-wrapper.doc-prop": {
```

Comment:
I think we'll have to change the architecture of the routes. To enable the doc prop to be rendered as an iframe, we need the routes to be named `docs-ui`. I don't have any good solutions this for now. But let's think about it!

Reply:
@juliarocha we are good to go.

Reply:
I'll let @saviomuniz approve when it is ready.